### DISCUSSION TITLE
Unable to register MSbuild defaults

### DISCUSSION THREAD

**Discussion by sn4k3** [EWSoftware/SHFB#937] [Q&A] [answered]
I'm with https://github.com/EWSoftware/SHFB/releases/tag/2022.8.14.1 installed and Visual Studio 2022 but when i try to open the Builder GUI i got this error message:

![image](https://user-images.githubusercontent.com/113281/190270516-3dd02d9e-6401-490a-840e-c484cd05302c.png)

My msbuild paths:

`C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\Bin\amd64\MSBuild.exe`
`C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\Bin\MSBuild.exe`

What can fix this?

**Comment by EWSoftware** [comment, marked as answer]
See the release notes.  A breaking change in the 17.3 version of MSBuild required a limitation on what the standalone GUI can use.  It will be removed in the next release as I found a workaround.  In the meantime, install the 2017 or 2019 build tools as indicated or use Visual Studio to manage the project as it has no restrictions.

**Reply by sn4k3** [reply]
Thanks 👍
